btree/btree.c

```diff
@@ -5082,19 +5110,19 @@ static void btree_ut_thread_kv_oper(struct btree_ut_thread_info *ti)
 			M0_ASSERT(data.flags == 0);
```

Comment:
you might want to use M0_BSC_SUCCESS instead of 0 as you have used M0_BSC_SUCCESS in some other places as well and it gives more readability.

Reply:
Good catch.